doc-content/kogito-docs/src/main/asciidoc/openshift/proc_kogito-deploying-on-ocp-console.adoc

```diff
@@ -0,0 +1,93 @@
+[id='proc_kogito-deploying-on-ocp-console']
+
+= Deploying {PRODUCT} services on {OPENSHIFT} using the OpenShift web console
+
+After you create your {PRODUCT} services as part of a business application, you can use the {OPENSHIFT} web console to deploy your services. As an aid to deploy your services, you can also use the {PRODUCT} Operator, which guides you through the deployment process. The {PRODUCT} Operator uses the https://github.com/operator-framework[Operator Framework] and automates many of the deployment steps for you. For example, you can provide the Operator a link to the Git repository that contains your application, and the operator can pull the project, build it (including native compilation, if applicable), and deploy the resulting services.
+
+.Prerequisites
+* The application with your {PRODUCT} services is in a Git repository.
+* You have access to the OpenShift web console with `cluster-admin` permissions.
```

Comment:
not sure if we need to emphasize user to use cluster-admin roles, most of developers that uses a pre installed cluster are not admins, maybe has permissions only to his namespace.

Reply:
To install the operator you need cluster-admin permissions. I believe we have a paragraph saying  something like: "if you don't have the rights make sure that the admin have installed the Kogito Operator for you".

Reply:
@spolti, what @ricardozanini said, correct.